autorest/codegen/models/parameter.py

```diff
@@ -194,6 +196,7 @@ def from_yaml(cls, yaml_data: Dict[str, Any]) -> "Parameter":
             constraints=[],  # FIXME constraints
             target_property_name=id(yaml_data["targetProperty"]) if yaml_data.get("targetProperty") else None,
             style=ParameterStyle(http_protocol["style"]) if "style" in http_protocol else None,
+            explode=http_protocol["explode"] if "explode" in http_protocol else False,
```

Comment:
tiniest nit: i think it'd be slightly cleaner to do `http_protocol.get("explode", False)`

Reply:
I try to remember why I did it like that, but I can't. I followed the other lines that were doing this. Will re-think stronger, thanks!